ndk/src/media/image_reader.rs

```diff
@@ -5,7 +5,8 @@
 #![cfg(feature = "api-level-24")]
 
 use super::NdkMediaError;
-use super::{construct, construct_never_null, error::MediaErrorResult, Result};
+use super::{construct, construct_never_null, Result};
+use crate::media_error::MediaErrorResult;
```

Comment:
These can still be imported from the same crate, right?

Reply:
They can, but If I put `use crate::media_error::MediaErrorResult` to `media/mod.rs` without `features = "api-level-24"`, `MediaErrorResult` is pointed out as an unused member. Should I put `pub use crate_media_error::MediaErrorResult` to `media/mod.rs`?

Reply:
In `media/mod.rs` it would have been `pub`.

Alas, since it seems we need to make a breaking change anyway (otherwise need a local `mod error { pub use super:...; }`) let's not reexport (`pub use`) anything.

Reply:
So you mean removing `pub use crate::media_error::{NdkMediaError, Result};` from `media/mod.rs` as well and reflect the changes to `media/{image_reader, media_codec}.rs`?

Reply:
Yup.

While at it, let's also update `ndk/CHANGELOG.md` to finalize this.

Reply:
Updated!